azure-pipelines/ui-automation/msal-local-nightly.yml

```diff
@@ -0,0 +1,159 @@
+  # MSAL with Broker Test Plan stage
+  - stage: 'msal_with_broker'
+    dependsOn:
+      - msalautomationapp
+      - brokers
+    displayName: Running MSAL without Broker Test UI Test Suite
+    jobs:
+      - job: 'msal_with_broker'
+        displayName: Running MSAL without Broker Test Plan
+        timeoutInMinutes: 90
+        pool:
+          vmImage: ubuntu-latest
+        variables:
+          resultsDir: "msal-BrokerHost-$(Build.BuildId)-$(Build.BuildNumber)"
+        steps:
+          - checkout: none
+          - task: DownloadSecureFile@1
+            displayName: 'Download Firebase Service Account Key File'
+            name: gcServiceAccountKey
+            inputs:
+              secureFile: AndroidFirebaseServiceAccountKey.json
+              retryCount: 5
+          - download: current
+          - script: gcloud version
+            displayName: 'Check gcloud version'
+          - task: Bash@3
+            displayName: Run UI Automation on Firebase
+            inputs:
+              targetType: inline
+              script: |
+                gcloud auth activate-service-account --key-file "$(gcServiceAccountKey.secureFilePath)"
+                gcloud config set project "$(gCloudProjectId)"
+                gcloud firebase test android run \
+                  --type instrumentation \
+                  --app "$(Pipeline.Workspace)/msalautomationapks/$(msalAutomationAppApk)" \
+                  --test "$(Pipeline.Workspace)/msalautomationapks/$(msalAutomationAppTestApk)" \
+                  --device "model=${{ parameters.firebaseDeviceId }},version=${{ parameters.firebaseDeviceAndroidVersion }}" \
+                  --timeout "$(firebaseTimeout)" \
+                  --other-files \
+                  "/data/local/tmp/AzureSample.apk=$(Pipeline.Workspace)/azureSample/$(azureSampleApk)" \
+                  --results-dir "$(resultsDir)" \
+                  --directories-to-pull "/sdcard" \
+                  --use-orchestrator \
+                  --environment-variables "clearPackageData=true" \
+                  --results-history-name "$(resultsHistoryName)" \
+                  --test-targets "notPackage com.microsoft.identity.client.msal.automationapp.testpass.perf, notPackage com.microsoft.identity.client.msal.automationapp.testpass.broker"
+          - script: gsutil cp "gs://test-lab-ffz6x9pu2y62a-is0rq7a7rwdhi/$(resultsDir)/${{ parameters.firebaseDeviceId }}-${{ parameters.firebaseDeviceAndroidVersion }}-en-portrait/test_result_1.xml" "$(Build.SourcesDirectory)"
+            displayName: Download Test Result File
+            condition: succeededOrFailed()
+          - task: PublishTestResults@2
+            displayName: Publish Test Results to ADO
+            condition: succeededOrFailed()
+            inputs:
+              testResultsFiles: '*test_result*.xml'
+              searchFolder: $(Build.SourcesDirectory)
+              testRunTitle: 'MSAL UI Automation - Build # $(Build.BuildNumber)'
```

Comment:
nit: add extra line

Reply:
Addressed here: https://github.com/AzureAD/microsoft-authentication-library-for-android/pull/1644/commits/b982fadf3654453e1c667d5ee6e933faac9aea8f